pkg/diagnostics/cluster/aggregated_logging/daemonsets.go

```diff
@@ -0,0 +1,118 @@
+package aggregated_logging
+
+import (
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapisext "k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+const daemonSetNoLabeledNodes = `
+There are no nodes that match the selector for DaemonSet '%[1]s'. This
+means Fluentd is not running and is not gathering logs from any nodes.
+An example of a command to target a specific node for this DaemonSet:
+
+  oc label node/ip-172-18-2-170.ec2.internal %[2]s
```

Comment:
I think this is a little *too* specific. Could it be something like:

    oc label node/node1.example.com %[2]s

Alternatively, I think we could leave the specific case out and just give the command to label all of them which is most likely to be what we want anyway.